pkg/resources/manager/functions.go

```diff
@@ -55,6 +60,11 @@ func SetMachineClasses(classes []*iri.MachineClass) error {
 	return mng.setMachineClasses(classes)
 }
 
+// SetVMLimit just registers maximum limit for VMs
+func SetVMLimit(vmCountLimit int) error {
```

Comment:
i am not big fan of type without size. Is possible use int32?

Reply:
changed it to uint64